core/vibes/soul/primitives/card-carousel/card-carousel.makeswift.tsx

```diff
@@ -0,0 +1,65 @@
+import { Image, Link, List, Select, Shape, Style, TextInput } from '@makeswift/runtime/controls';
+
+import { runtime } from '~/lib/makeswift/runtime';
+
+import { CardCarousel } from '.';
+
+interface Card {
+  title?: string;
+  imageSrc?: string;
+  imageAlt?: string;
+  link?: { href?: string; target?: string };
+}
+
+interface MSCardCarouselProps {
+  className: string;
+  textContrast: 'light' | 'dark';
```

Comment:
I'm not seeing an difference when toggling the text contrast from light to dark. Should there be?

Reply:
Fixed